backend/backend/DTOs/UserDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace backend.DTOs
+{
+    public class UserDto
+    {
+        public Guid id { get; set; }
+        public string firstname { get; set; }
+        public string lastname { get; set; }
+        public string email { get; set; }
```

Comment:
Same here, PascalCase for property names

Reply:
Done